clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/handler/KubernetesStatefulSetHandler.java

```diff
@@ -144,6 +145,19 @@ private Status status(V1beta2StatefulSet statefulSet) {
       return result.unstable("Waiting for all updated replicas to be ready");
     }
 
+    String updateType = statefulSet.getSpec().getUpdateStrategy().getType();
+    V1beta2RollingUpdateStatefulSetStrategy rollingUpdate = statefulSet.getSpec().getUpdateStrategy().getRollingUpdate();
+    if (updateType.equalsIgnoreCase("rollingupdate")) {
```

Comment:
is this actually case insensitive or is IgnoreCase being used out of an (over?) abundance of caution here? If it does need to be case insensitive, do we have a test case for `rollingUpdate` somewhere?

Reply:
100% out of caution. Pretty sure it's camelCased. Don't care to remove this.

Reply:
Misunderstood what you meant previously, this is case-insensitive, we don't need to worry about other cases.